### PR TITLE
Fixed BUG 3612 

### LINKED ISSUE #3612: [BUG] Live Stream Review Template (404 NOT FOUND)
### Description of the bug

The "Live Stream Review Template" page is not found, even though it's mentioned in the side as a catergory.

### Project stack or language (e.g. front-end)

front-end, maybe?

### Working Environment (e.g. operating system, browser, device)

OS: Windows 10
Browser: Chrome
Device: PC

### Please add screenshots (if applicable)

image

### Add any other context about the problem here

_No response_

### PR DESCRIPTION
**Pull request by alishashaw439**
closes #3612

The links of the options mentioned in the sidebar.md file was not working. Now it will work.

### CODE REVIEW

**Comment by alishashaw439** [comment]
@eddiejaoude I am new in contributing to open source. Kindly review this PR. Thanks!

File: docs/_layout/sidebar.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+  - [Git Tips and Tricks](../tips/git.md)
+  - [How to Use GitHub](../tips/githubtips.md)
+  - [Common Terms](../tips/glossary.md)
+  - [Open Source Resources](../tips/OpenSourceResources.md)
+  - [Open Source Tips](../tips/OpenSourceTips.md)
+  - [Glossary](../tips/glossary.md)
 
 - Templates
-  - [YouTube Generic Template](/youtube/generic.md)
-  - [LiveStream Review Template](/youtube/livestream.md)
```
Comment (eddiejaoude) [inline docs/_layout/sidebar.md:23]: I don't think these paths need changing?

File: docs/youtube/livestream.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Moderating Youtube
+
+Coming soon! :)
```
Comment (eddiejaoude) [inline docs/youtube/livestream.md:3]: Probably better to delete the link to this file, rather than create a file

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
I made inline comments

File: docs/_layout/sidebar.md (hunk 1 above)
Comment (alishashaw439) [reply docs/_layout/sidebar.md:23]: The md files are outside the _layout folder. If it is not being used in the website then it should not be changed.

File: docs/youtube/livestream.md (hunk 2 above)
Comment (alishashaw439) [reply docs/youtube/livestream.md:3]: I'll do it.

File: docs/_layout/sidebar.md (hunk 1 above)
Comment (eddiejaoude) [reply docs/_layout/sidebar.md:23]: The files are outside the folder but when the app is built and deployed, I think it is all working fine https://eddiehubcommunity.github.io/support/#/discord/discord-readme

File: docs/youtube/livestream.md (hunk 2 above)
Comment (eddiejaoude) [reply docs/youtube/livestream.md:3]: Thanks 👍

File: docs/_layout/sidebar.md (hunk 1 above)
Comment (alishashaw439) [reply docs/_layout/sidebar.md:23]: Thanks for pointing that out! I have removed my changes. Other than that, I have removed the link of the livestream.md file.

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for making the changes 👍 

It looks good, but CI is failing due to a formatting error
```
./docs/_layout/sidebar.md:22:51 MD047/single-trailing-newline Files should end with a single newline character
```

**Comment by alishashaw439** [comment]
> Thank you for making the changes 👍
> 
> It looks good, but CI is failing due to a formatting error
> 
> ```
> ./docs/_layout/sidebar.md:22:51 MD047/single-trailing-newline Files should end with a single newline character
> ```

I made some changes. Now it is not showing any lint error. Thank you for your patience.

**Review by eddiejaoude — APPROVED** [review]
Looks great, thank you 👍

**Pull request merged**